pkg/sbom/scanner/scanner.go

```diff
@@ -139,6 +139,17 @@ func (s *Scanner) enoughDiskSpace(opts sbom.ScanOptions) error {
 		return fmt.Errorf("not enough disk space to safely collect sbom, %d available, %d required", usage.Available, opts.MinAvailableDisk)
 	}
 
+	if imgMeta == nil || opts.OverlayFsScan || opts.UseMount {
```

Comment:
Shouldn't this be move on top of the previous check ?

Reply:
I wanted to keep the ability to have a minimal size check to handle things like cache. The default is pretty small (1G)